test.sh

```diff
@@ -0,0 +1,8 @@
+export ARM_SUBSCRIPTION_ID=67a9759d-d099-4aa8-8675-e6cfd669c3f4
+export ARM_CLIENT_ID=6fe4b504-6bd7-4cce-b113-a663e4050ab6
+export ARM_CLIENT_SECRET=83287e7a-e5a9-4f46-a646-77b71930a64b
+export ARM_TENANT_ID=72f988bf-86f1-41af-91ab-2d7cd011db47
+export ARM_TEST_LOCATION=EastUS2
+export ARM_TEST_LOCATION_ALT=EastUS
+export ARM_TEST_LOCATION_ALT2=WESTUS2
+make testacc TESTARGS="-run=TestAccAzureRMSubnetNatGatewayAssociation_deleted"
```

Comment:
Is this file getting checked in by mistake? Typically we don't check-in our subscription information.

Reply:
My mistake...sorry about this.